perf/benchmark/run_benchmark_job.sh

```diff
@@ -74,30 +83,63 @@ pipenv install
 pushd "${WD}"
 ./setup_test.sh
 popd
+dt=$(date +'%Y%m%d-%H')
+export OUTPUT_DIR="benchmark_data.${GIT_SHA}.${dt}"
```

Comment:
We should switch date and sha order here so that files can be ordered.

Reply:
ok done